hw_6/script.js

```diff
@@ -0,0 +1,89 @@
+//prettier-ignore
+let initialArray = [
```

Comment:
It is better to use the `const` in places where it is possible. Check out your code.

Reply:
https://github.com/BlackTruth/javascript/blob/26e480b6e9d9e22ba634d5513e6e430222b0590e/hw_6/script.js#L2-L19
Of course, as always... Fixed.